bech32m/codecs.py

```diff
@@ -133,7 +134,7 @@ def convertbits(data, frombits, tobits, pad=True):
     return ret
 
 
-def decode(hrp, addr):
+def decode(hrp: str, addr: str) -> tuple[int, bytes]:
```

Comment:
Should I replace returning type with `namedtuple`? If so, what name for type and fields do you prefer? It's compatible change.

Reply:
Done